src/utils.ts

```diff
@@ -0,0 +1,35 @@
+export type Donation = {
+  amount: number;
+  currencyCode: string;
+  receiptUrl: string;
+};
+
+export async function getDonations(): Promise<Donation[]> {
+  try {
+    const response = await fetch(
+      'https://donations-api.justgiving.com/v1/donations',
+      {
+        headers: {
+          'X-Secret-Key': 'eromynAterceSoStoN',
+        },
+      },
+    );
+
+    return response.json();
+  } catch (err) {
+    return [];
+  }
+}
+
+export function track(event: string) {
+  (window as any).track(event);
+}
+
+export function formatCurrency(amount: number, currencyCode: string) {
+  const codeToSymbol: Record<string, string> = {
+    GBP: '£',
+    USD: '$',
+  };
+
+  return amount.toFixed(2) + codeToSymbol[currencyCode];
```

Comment:
seems odd to have the currency after the amount. Also currencyCode could be any string and the object could return undefined